packages/@ember/-internals/glimmer/lib/syntax/in-element.ts

```diff
@@ -27,8 +27,16 @@
  It also supports an optional named argument:
 
  - `insertBefore` -- by default the DOM element's content is replaced when used as
- `destinationElement`. Passing `null` changes the behaviour to appended at the end
- of any existing content. Any other value than `null` is currently not supported.
+ `destinationElement`. Passing `null` to `insertBefore` changes the behaviour to
+ append the block content to the end of any existing content. Any other value than
+ `null` is currently not supported.
+
+ For example:
+ ```handlebars
+ {{#in-element this.destinationElement insertBefore=null}}
+  <div>Some content</div>
+ {{/in-element}}
+ ```
```

Comment:
Markdown formatting:

```suggestion
     For example:

     ```handlebars
     {{#in-element this.destinationElement insertBefore=null}}
       <div>Some content</div>
     {{/in-element}}
     ```
```

This will make it part of the actual bullet point describing the argument, and will also make sure the code block is properly parsed as distinct from the preceding comment regardless of the Markdown parser used.